client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -75,6 +75,24 @@ Undocumented.prototype.accountRecoveryReset = function( userData ) {
 };
 
 /*
+ * Verify a wpcom user credentials.
+ *
+ * @param {string} [username_or_email]
+ * @param {string} [password]
+ * @param {Function} fn
+ * @api public
+ */
+Undocumented.prototype.verifyLogin = function( username_or_email, password, fn ) {
```

Comment:
As we talked about, we aren't going to be using this endpoint, so we probably shouldn't merge code for it. What do you think about:

- Removing the change to this file.
- Changing the submit handler to a `noop` in `Login`.
- Removing the new actions/tests for those actions.

and then adding new actions/tests/wpcom function for the new endpoint in a future PR?